### PR TITLE
replace print with verbose logging

### PR DESCRIPTION
**Pull request by codexp**
There were some annoying server console prints of `"lrfurn:armchair_"..color` and some other print calls, I assume for debugging.
Those were replaced with `minetest.log("verbose", ...` to respect log level setting and not clutter server console.
Also `[HomeDecor] loaded!` log has been placed in a condition as in many other mods too.

### CODE REVIEW

**Comment by VanessaE** [comment]
I think it would be better to omit the debug output altogether.

File: homedecor/climate-control.lua [hunk 1]
```diff
@@ -25,7 +25,7 @@ minetest.register_entity("homedecor:mesh_desk_fan", {
 })
 
 local add_mesh_desk_fan_entity = function(pos)
-	print("in add_mesh_desk_fan_entity()")
+	minetest.log("verbose", "in add_mesh_desk_fan_entity()")
```
Comment (codexp) [inline homedecor/climate-control.lua]: I think this should be removed.

File: homedecor/furniture.lua [hunk 2]
```diff
@@ -235,7 +235,7 @@ minetest.register_lbm({
 		local param2 = paletteidx + new_fdir
 		local newname = "homedecor:armchair"
 
-		print(name, dump(a), dump(b), dump(color).."("..dump(paletteidx)..")", dump(param2))
+		minetest.log("verbose", name, dump(a), dump(b), dump(color).."("..dump(paletteidx)..")", dump(param2))
```
Comment (codexp) [inline homedecor/furniture.lua]: this should be removed

**Comment by codexp** [comment]
I've removed debug logging.

**Pull request merged**